app/src/main/res/layout/activity_dashboard.xml

```diff
@@ -1,407 +1,517 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
804 lines of diff 🤯 . Can you please try to remove the unneeded changes from this file?

Reply:
I actually changed the color scheming of all the cards, added 2 new ones for Critique and Listen, hence the lines fluctuations! :)